cst438/src/main/java/cst438/controller/CityRestController.java

```diff
@@ -27,4 +34,16 @@
 
 		return new ResponseEntity<List<City>>(cities, HttpStatus.OK);
 	} 
+	
+	@PostMapping("/api/cityFromCountry")
+	public ResponseEntity<?> getCountry(@RequestParam("countryName") String countryName) {
+		List<City> cities = cityRepository.findAllByOrderByNameAsc(countryName);
```

Comment:
if we make the change per my other comment, this also needs to change to use `findAllByCountryCodeOrderByNameAsc(countryCode);`